cmd/strided/config_defaults.go

```diff
@@ -0,0 +1,262 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	tmcli "github.com/cometbft/cometbft/libs/cli"
+	"github.com/cosmos/cosmos-sdk/server"
+)
+
+/*
+	This file is intended to replace defaults in config.toml and app.toml
+	by overwriting the values in the respective files.
+
+	The actions of this file can be disabled with the --reject-config-defaults flag.
+
+	This file is taken almost entirely from the Osmosis repo, with minor modifications.
+	Their original file can be found here:
+	https://github.com/osmosis-labs/osmosis/blob/e5895ce4a460a585c0afb29873de9c7de826b690/cmd/osmosisd/cmd/root.go#L1
+	Thank you to the excellent engineers at Osmosis.
+*/
+
+const (
+	FlagRejectConfigDefaults = "reject-config-defaults"
+)
+
+type SectionKeyValue struct {
+	Section string
+	Key     string
+	Value   any
+}
+
+var (
+	recommendedAppTomlValues = []SectionKeyValue{
+		{
+			Section: "",
+			Key:     "minimum-gas-prices",
+			Value:   "0.0005ustrd,0.001stuosmo,0.0001stuatom,20000000000staevmos,500000000stinj,0.01stutia,15000000000stadydx,15000000000stadym,0.01stusaga,0.001ibc/D24B4564BCD51D3D02D9987D92571EAC5915676A9BD6D9B0C1D0254CB8A5EA34,0.0001ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2,0.01ibc/BF3B4F53F3694B66E13C23107C84B6485BD2B96296BB7EC680EA77BBA75B4801,20000000000ibc/4B322204B4F59D770680FE4D7A565DDC3F37BFF035474B717476C66A4F83DD72,500000000ibc/A7454562FF29FE068F42F9DE4805ABEF54F599D1720B345D6518D9B5C64EA6D2,15000000000ibc/561C70B20188A047BFDE6F9946BDDC5D8AC172B9BE04FF868DFABF819E5A9CCE,15000000000ibc/E1C22332C083574F3418481359733BA8887D171E76C80AD9237422AEABB66018,0.01ibc/520D9C4509027DE66C737A1D6A6021915A3071E30DBA8F758B46532B060D7AA5",
+		},
+	}
+
+	recommendedConfigTomlValues = []SectionKeyValue{
+		{
+			Section: "consensus",
+			Key:     "timeout_commit",
+			Value:   "1s",
+		},
+	}
+)
+
+// overwriteConfigTomlValues overwrites config.toml values. Returns error if config.toml does not exist
+//
+// Currently, overwrites:
+// - timeout_commit
+//
+// Also overwrites the respective viper config value.
+//
+// Silently handles and skips any error/panic due to write permission issues.
+// No-op otherwise.
+func overwriteConfigTomlValues(serverCtx *server.Context) error {
```

Comment:
nit: are the config and app toml functions identical besides the reference to "config.toml/app.toml" and "recommendedConfigTomlValues/recommendedAppTomlValues"? If so, might make sense to consolidate the functions